references.go

```diff
@@ -10,15 +10,18 @@ import (
 	"github.com/sergi/go-diff/diffmatchpatch"
 )
 
-// References returns a References for the file at "path", the commits are
-// sorted in commit order. It stops searching a branch for a file upon reaching
-// the commit were the file was created.
+// References returns a slice of Commits for the file at "path", starting from
+// the commit provided that contains the file from the provided path. The last
+// commit into the returned slice is the commit where the file was created.
+// If the provided commit does not contains the specified path, a nil slice is
+// returned. The commits are sorted in commit order, newer to older.
 //
 // Caveats:
+//
 // - Moves and copies are not currently supported.
+//
 // - Cherry-picks are not detected unless there are no commits between them and
-//   therefore can appear repeated in the list.
-//   (see git path-id for hints on how to fix this).
+// therefore can appear repeated in the list. (see git path-id for hints on how to fix this).
```

Comment:
This comment line looks longer than the rest, it may be longer than 60 or 70 chars (or whatever the line length we are using for comments).